tests/tests.rs

```diff
@@ -372,6 +372,20 @@ fn port_check() {
     assert_eq!(page.header.status, Status::ProxyRequestRefused);
 }
 
+#[test]
+/// - port is not checked if the skip option is passed.
+/// - status for wrong port is "proxy request refused"
+fn port_check_skipped() {
+    let page = get(
+        &["--addr", "[::]:19720", "--hostname", "example.org", "--skip-port-check"],
+        addr(19720),
+        "gemini://example.org:1971/",
+    )
+    .expect("could not get page");
+
+    assert_eq!(page.header.status, Status::ProxyRequestRefused);
```

Comment:
If the port check is skipped, the result would be `Status::Success`, because `Status::ProxyRequestRefused` can only be returned if the port check is active.

Reply:
Oops sorry for that, should have not pushed right before bedtime!